.codeclimate.yml

```diff
@@ -1,28 +1,26 @@
 ---
-engines:
+version: "2"
+checks:
+  method-complexity:
+    config:
+      threshold: 20
```

Comment:
(This is `5` by default, which is crazy.)

Reply:
I think 5 -> 20 is a bit of a drastic change. What do you think about bumping it to 10 and seeing how we like it there?

Reply:
Honestly, I'm tempted to remove the check entirely. The "complexity" check is something that CodeClimate added last month, so the whole repo was written without such a check in mind. And, like I said, the check is so strict that kinda it makes all but the most trivial functions impossible to write.

I think setting the threshold to 20 (or even higher) is necessary to make the check accomplish what it's supposed to do – namely, discourage writing methods that are difficult to reason about.